operator/pkg/util/patcher/pather.go

```diff
@@ -47,6 +68,33 @@ func (p *Patcher) ForDeployment(deployment *appsv1.Deployment) {
 
 	deployment.Annotations = labels.Merge(deployment.Annotations, p.annotations)
 	deployment.Spec.Template.Annotations = labels.Merge(deployment.Spec.Template.Annotations, p.annotations)
+
+	if len(p.extraArgs) != 0 || len(p.featureGates) != 0 {
+		// It's considered the first container is the karmada-etcd by default.
```

Comment:
karmada-etcd? This comment is a little confusing.

Reply:
Yeah, this is a bad  describe, updated.